type.go

```diff
@@ -397,6 +408,183 @@ func NewFreeFloating(ve Vehicle) (*FreeFloating) {
 	}
 }
 
+// Structure and Consumer to creates StopPoint objects based on a line read from a CSV
+type StopPoint struct {
+	Id 			string
+	Name 		string
+	Sens		int
```

Comment:
```suggestion
	Direction	int  
```
and add a comment to state what `Direction` represents ?